template.yaml

```diff
@@ -37,6 +37,33 @@ Resources:
         CertificateArn: !Ref AcmCertificateArn
         EndpointConfiguration: EDGE
 
+  OutputBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      OwnershipControls:
+        Rules:
+          - ObjectOwnership: ObjectWriter
+      PublicAccessBlockConfiguration:
+        BlockPublicPolicy: False
+      AccessControl: PublicRead
+
+  OutputBucketPublicRead:
```

Comment:
adding `Policy` to the resource name would help.  Also I don't think you even need to setup a policy if create `OutputBucket` using the example in the s3 docs.